Source/Core/DolphinQt/Config/Mapping/MappingNumeric.cpp

```diff
@@ -15,6 +15,8 @@ MappingDouble::MappingDouble(MappingWidget* parent, ControllerEmu::NumericSettin
   setRange(m_setting.GetMinValue(), m_setting.GetMaxValue());
   setDecimals(2);
 
+  setFixedWidth(112);
```

Comment:
While it looks good visually, where does 112 come from exactly? It sounds very arbitrary and is bound to raise questions.

Reply:
112 is taken from: https://github.com/dolphin-emu/dolphin/blob/master/Source/Core/DolphinQt/Config/Mapping/MappingButton.cpp#L52
Which doesn't have any explanation as to why that exact width is chosen.
I could add a comment referencing the value in MappingButton.cpp.

Reply:
It would then be better to make this a constexpr value somewhere in `Mapping` headers and reference it in both places.

Reply:
Will do.

Reply:
I made it into a constexpr but I'm not sure if this is the correct way to handle it.

Reply:
Looks good to me!